src/server.js

```diff
@@ -64,7 +64,19 @@ const migrate = async (isApi) => {
  */
 const createServer = async (port, isApi) => {
   const server = await new Hapi.Server({
-    port
+    port,
+    routes: {
+      validate: {
+        options: ErrorHandler.validateRoutes(),
+        failAction: async (request, h, err) => {
+          throw ErrorHandler.Factory.createFSPIOPErrorFromJoiError(err)
```

Comment:
How do you know that this was a JOI error?

I suspect you need to rather use `throw Boom.boomify(err)`

See example --> https://github.com/mojaloop/ml-api-adapter/blob/master/src/shared/setup.js#L59